frontend/src/css/modules/base.scss

```diff
@@ -1006,16 +1006,22 @@ h4.challenge-step-title {
         background: white;
     }
 
+    :root {
+        .col.m3 {
```

Comment:
@Alabhya268 why is this done just for `col m3` we will need it for other column sizes too, no?

Reply:
@Ram81 , 'col m3'  is used for challenge cards, Therefore it is unecessary to change sizes for other columns.